### PR TITLE
Customize/colors : Move Orange's colors paragraph and add warning

### LINKED ISSUE #1356: Docs > Customize > Color : Change information order in the page + add explanations
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

**To be corrected**:
- docs/v5.2/customize/color: 
    - Put the paragraph "Orange's colors" at the very beginning of the page, as the 1st paragraph of the page
    - In the paragraph "All colors", put the text "All Boosted colors are available as Sass variables and a Sass map..." in an ODS **warning** message.

### Motivation and context

This issue comes from the work done in Issue https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1201 List in all the doc the incompatibility with ODS in which all Boosted components (variants included) have been inspected in order to identify the ones that do not respect ODS specifications and therefore need an alert message saying "Do not use in ODS".
Doing this work, design bugs/corrections to do were noticed.

### PR DESCRIPTION
**Pull request by hannahiss**
### Related issues

Closes #1356 

### Description
- Put the paragraph "Orange's colors" at the very beginning of the page, as the 1st paragraph of the page
- In the paragraph "All colors", put the text "All Boosted colors are available as Sass variables and a Sass map..." in an ODS warning message.

### Motivation & Context
This issue comes from the work done in Issue #1201 

### Types of change
Update doc only

### Live previews

* https://deploy-preview-1610--boosted.netlify.app/docs/5.2/customize/color/

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [ ] My change introduces changes to the migration guide
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by hannahiss** [comment]
It is to be noted that the sub-section"Notes on Sass" that is in Bootstrap under "All colors" paragraph, is under "Orange's colors" in this version (I didn't change it but I mention it to be sure it is wanted).

File: site/content/docs/5.2/customize/color.md [hunk 1]
```diff
@@ -225,6 +152,83 @@ Sass cannot programmatically generate variables, so we manually created variable
...
+</div>
+
+All these colors are available as a Sass map, `$theme-colors`.
+
+{{< scss-docs name="theme-colors-map" file="scss/_variables.scss" >}}
+
+Check out [our Sass maps and loops docs]({{< docsref "/customize/sass#maps-and-loops" >}}) for how to modify these colors.
+
+## All colors
+<div class="bd-callout bd-callout-warning small">
```
Comment (louismaximepiton) [inline site/content/docs/5.2/customize/color.md:178]: IMO, we should use `{{< callout warning >}}` here. So we need to have the `.small` class ?

**Review by louismaximepiton — APPROVED** [review]
We should the a11y team to have their point of view.

File: site/content/docs/5.2/customize/color.md (hunk 1 above)
Comment (julien-deramond) [reply site/content/docs/5.2/customize/color.md]: Agree. I've made a fix for that.

**Review by julien-deramond — APPROVED** [review]
Made some changes to reflect the comments in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1610/commits/784bccaa409763ea34cb678287b58f83888134e2 and https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1610/commits/56da219eb359723988a382ae013b2b24b184ce9b

**Pull request merged**